index.html

```diff
@@ -102,9 +102,124 @@ <h2>Work</h2>
                 <h2>MY RECENT WORK</h2>
             </div>
           </article>
-         
-        </div>   
-    </main>
+
+          <article class="down">
+            <div class="box-1">
+                <img src="./images/ImgPlaceholder.png" alt="Blank square">
+                <div class="box-2">
+                    <h1>Multi-Post Stories</h1>

```

Comment:
- [ ] Please know that you should wrap the entirety of the recent works section in a `section` tag, as per the requirements.